livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterpreter.java

```diff
@@ -119,25 +144,25 @@ public void close() {
     }
   }
 
-  protected void initLivySession() throws LivyException {
```

Comment:
initLivySession() was setting `this.sessionInfo`. 

changed name to createSession and removed assignment to`this.sessionInfo` to its caller